frontend/tailwind.config.js

```diff
@@ -14,7 +14,12 @@ module.exports = {
       darkbrown: "#513131",
       white: "#FFFFFF",
     },
-    extend: {},
+
```

Comment:
maybe use a regular image and not a background image. Otherwise you cannot show separate pics for recipes